src/http/actions/actions.dependents.ts

```diff
@@ -250,4 +250,24 @@ export module DependentsActions {
       callback(null, dependent);
     });
   };
+   /**
+   * ! Method to delete dependent by ID
+   * * Alcazar87 - 2021/12/30
+   * @param dependentID {string}
+   * @param callback {Function}
+   */
+    export const deleteDependent = (dependentID: string, callback: Function) => {
+      const queryString = `
+      DELETE 
+      FROM ${_dependentTableName}
+      INNER JOIN ${_customerDependentRealtionshipTableName} 
+      ON ${_dependentTableName}.id = ${_customerDependentRealtionshipTableName}.dependentID 
+      WHERE id=${dependentID}`;
```

Comment:
@Alcazar87 This action is used by '/my-dependents/:dependentID' endpoint who must grant only to delete owned dependents for loged in customers.
No controler or actions verifies that dependent is owned by cuntomer calling endpoint, so any logged customer can delete any dependent.
I think we must write two separated "chains" endpoint -> controller -> action for admins and customers. The exisitng action is good for admins with a ".deleteAny" permission, but we need a new one for customers with ".deleteOwn" permissions.